test/composite.test.js

```diff
@@ -11,5 +11,5 @@ test('anonymous function should be named composite.test', t => {
     next()
   })
 
-  t.is(fn[Symbol.for('fastify.display-name')], 'composite.test')
+  t.is(fn[Symbol.for('plugin-meta')].name, 'composite.test')
```

Comment:
Can you name this `fastify.plugin-meta`?

Reply:
If the symbol is changed, [this line](https://github.com/fastify/fastify/blob/68114ee8b4a46ebb637d96ea3dbdea28f0cf94ca/lib/pluginUtils.js#L7) in the `fastify` repo would also need to change, so changing the symbol could be considered a breaking change.

Reply:
Ouch, forget about it!